gno.land/pkg/gnoland/node_inmemory.go

```diff
@@ -127,29 +123,3 @@ func NewInMemoryNode(logger *slog.Logger, cfg *InMemoryNodeConfig) (*node.Node,
 		logger,
 	)
 }
-
-// GetNodeReadiness waits until the node is ready, signaling via the EventNewBlock event.
-// XXX: This should be replace by https://github.com/gnolang/gno/pull/1216
-func GetNodeReadiness(n *node.Node) <-chan struct{} {
```

Comment:
@gfanton do you confirme we can remove this?

Reply:
It's unused, not sure how it wasn't caught by the linter